src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -52,14 +74,40 @@ public UserTenant getById(UUID consortiumId, UUID id) {
     return converter.convert(userTenantEntity, UserTenant.class);
   }
 
+  @Override
+  public UserTenant save(UUID consortiumId, UserTenant userTenantDto) {
+    String currentTenantId = HelperUtils.getTenantId(folioExecutionContext);
+    consortiumService.checkConsortiumExistsOrThrow(consortiumId);
+
+    Optional<UserTenantEntity> userTenant = userTenantRepository.findByUserIdAndTenantId(userTenantDto.getUserId(), IS_PRIMARY_TRUE);
+      if (userTenant.isEmpty()) {
+        throw new ResourceNotFoundException(USER_ID, String.valueOf(userTenantDto.getUserId()));
+      } else {
+      UserTenantEntity userTenantEntity = userTenant.get();
+       if (Boolean.FALSE.equals(userTenantEntity.getIsPrimary())) {
```

Comment:
what is expected behavior for userTenantEntity.getIsPrimary()==null? Shouldn't there be error as well?

Reply:
we wont be getting null , because we are querying the userTenant based on isPrimary and tenantId . so i think no need to test userTenantEntity.getIsPrimary()==null. Please suggest.

Reply:
ok then